### ISSUE TITLE
Make trailing_comma rule autocorrectable

### ISSUE THREAD

**Issue by wongzigii** [realm/SwiftLint#1326]
When comma is nested with Dictionary,  autocorrect command will not correct properly.

````swift
let inner = JSON([
     "some_field": "1" + "2",  // warnings
])



let inner = [
     "some_field": "1" + "2", // warnings
]
````

Expect:
````swift
let inner = [
     "some_field": "1" + "2"
]
````

**Comment by sammy-SC** [comment]
@wongzigii Hello,

I believe `trailing_comma` rule doesn't support `autocorrect`.

**Comment by marcelofabri** [comment]
I'll reopen this to track adding support to `autocorrect` to this rule.

**Comment by sammy-SC** [comment]
@marcelofabri I would like to get on it if no one else is working on it.

**Comment by marcelofabri** [comment]
I don't think anyone is working at it right now, so feel free to grab it! 💯 

Just a reminder: this rule supports two styles according to the configuration, so this should be respected when implementing autocorrect.

**Comment by marcelofabri** [comment]
Fixed on #1335 🎉
